src/ocm/components/hosts/use-hosts-table.tsx

```diff
@@ -140,6 +140,10 @@ export const useHostsTable = (cluster: Cluster) => {
         }
 
         const { data } = await HostsService.updateDiskRole(host, diskId, role);
+        //once selecting a disk which is in role none as a installation disk , the skip formatting attribute should be unselected and set to false
+        if (role === 'install') {
```

Comment:
If we end up with this solution on the UI rather than on the API, then I suggest we use the same PATCH to send all the parameters rather than having to perform two separate patches. 

It can even be part of the `updateDiskRole` function.